modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace by deleting the `FIM` resources.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+To remove the File Integrity Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. Delete all `FileIntegrity` objects, SELinux profiles, and webhook configurations.
```

Comment:
Could you remove the  SELinux profiles, and webhook configurations part, I think they only need to keep ` Delete all `FileIntegrity` objects`